e-digi/src/controller/Main.java

```diff
@@ -14,5 +15,14 @@ public static void main(final String[] args) throws Exception {
         autoradd.addAutor(autor);
         autoradd.addAutor(autor1);
 
+        // criando categoria
+        Categoria categoria = new Categoria("Suspense");
+        Categoria categoria1 = new Categoria("Ficção");
+
+        // realizando as inserções e imprimindo no caso de sucesso
+        DaoCategoria categoriaadd = new DaoCategoria();
+        categoriaadd.addCategoria(categoria);
+        categoriaadd.addCategoria(categoria1);
```

Comment:
Recomendo ter um main para cada coisa que for testar, vai facilitar depois, mas pode continuar assim se preferir.